### ISSUE TITLE
cannot install herokuish deb with docker-ce deb

### ISSUE THREAD

**Issue by znz** [dokku/dokku#2761]
Description of problem:

`apt install herokuish` error with docker-ce deb.

Output of the following commands

```
vagrant@vagrant:~$ dokku report
-----> uname: Linux vagrant 3.13.0-103-generic #150-Ubuntu SMP Thu Nov 24 10:34:17 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
-----> memory:
             total       used       free     shared    buffers     cached
Mem:           993        813        180          0         51        576
-/+ buffers/cache:        184        809
Swap:         1023          0       1023
-----> docker version:
Client:
 Version:      17.03.1-ce
 API version:  1.27
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Mon Mar 27 17:10:36 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.03.1-ce
 API version:  1.27 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Mon Mar 27 17:10:36 2017
 OS/Arch:      linux/amd64
 Experimental: false
-----> docker daemon info:
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 1
Server Version: 17.03.1-ce
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 4
 Dirperm1 Supported: false
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 4ab9917febca54791c5f071a9d1f404867857fcc
runc version: 54296cf40ad8143b62dbcaa1d90e520a2136ddfe
init version: 949e6fa
Security Options:
 apparmor
Kernel Version: 3.13.0-103-generic
Operating System: Ubuntu 14.04.5 LTS
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 993.8 MiB
Name: vagrant
ID: BNOD:BUFY:JZCV:IDCP:B6XL:RDWP:VWZM:ICHC:BATD:RMEC:LWE6:JZTB
Docker Root Dir: /var/lib/docker
Debug Mode (client): true
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
-----> sigil version: 0.4.0
-----> herokuish version:
Unable to find image 'gliderlabs/herokuish:latest' locally
latest: Pulling from gliderlabs/herokuish
c02c7df4a131: Already exists
a3ed95caeb02: Already exists
b16a7286d2a5: Already exists
011351573868: Already exists
aa35f6dee447: Pull complete
e2602cf3145d: Pull complete
f9a405a9f685: Pull complete
56172670ece7: Pull complete
Digest: sha256:62519f10682fa791f9b02e21d4e86969e2d6f1bd2bc5fc18bbfbf6aebc3a6e80
Status: Downloaded newer image for gliderlabs/herokuish:latest
herokuish: 0.3.29
buildpacks:
  heroku-buildpack-multi     v1.0.0
  heroku-buildpack-ruby      v159
  heroku-buildpack-nodejs    v99
  heroku-buildpack-clojure   v76
  heroku-buildpack-python    v99
  heroku-buildpack-java      v51
  heroku-buildpack-gradle    v21
  heroku-buildpack-grails    v21
  heroku-buildpack-scala     v76
  heroku-buildpack-play      v26
  heroku-buildpack-php       v121
  heroku-buildpack-go        v64
  heroku-buildpack-erlang    fa17af9
  buildpack-nginx            v8
-----> dokku version: 0.9.4
-----> dokku plugins:
 !     Deprecated: Please use plugin:list
plugn: 0.3.0
  00_dokku-standard    0.9.4 enabled    dokku core standard plugin
  20_events            0.9.4 enabled    dokku core events logging plugin
  apps                 0.9.4 enabled    dokku core apps plugin
  build-env            0.9.4 enabled    dokku core build-env plugin
  certs                0.9.4 enabled    dokku core certificate management plugin
  checks               0.9.4 enabled    dokku core checks plugin
  common               0.9.4 enabled    dokku core common plugin
  config               0.9.4 enabled    dokku core config plugin
  docker-options       0.9.4 enabled    dokku core docker-options plugin
  domains              0.9.4 enabled    dokku core domains plugin
  enter                0.9.4 enabled    dokku core enter plugin
  git                  0.9.4 enabled    dokku core git plugin
  logs                 0.9.4 enabled    dokku core logs plugin
  named-containers     0.9.4 enabled    dokku core named containers plugin
  nginx-vhosts         0.9.4 enabled    dokku core nginx-vhosts plugin
  plugin               0.9.4 enabled    dokku core plugin plugin
  proxy                0.9.4 enabled    dokku core proxy plugin
  ps                   0.9.4 enabled    dokku core ps plugin
  repo                 0.9.4 enabled    dokku core repo plugin
  shell                0.9.4 enabled    dokku core shell plugin
  ssh-keys             0.9.4 enabled    dokku core ssh-keys plugin
  storage              0.9.4 enabled    dokku core storage plugin
  tags                 0.9.4 enabled    dokku core tags plugin
  tar                  0.9.4 enabled    dokku core tar plugin
```

Environment details (AWS, VirtualBox, physical, etc.): vagrant with VirtualBox

How was Dokku installed?: manually from apt

How reproducible:

Steps to Reproduce:
1. install docker-ce with viewing https://docs.docker.com/engine/installation/linux/ubuntu/
2. install dokku with viewing http://dokku.viewdocs.io/dokku/#install-apt
3. `sudo apt install herokuish`

Actual Results:

```
vagrant@vagrant:~$ sudo apt install herokuish
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 herokuish : PreDepends: docker-engine-cs (>= 1.9.1) but it is not installable or
                         docker-engine (>= 1.9.1) but it is not installable
E: Unable to correct problems, you have held broken packages.
```

Expected Results:
Install successfully.

Additional info:
- Related: https://github.com/dokku/dokku/commit/1e387b3b420a49ff87d16265a30efade2f1a0dbf
- Redirected from https://github.com/gliderlabs/herokuish/issues/255

**Comment by josegonzalez** [comment]
I've released herokuish 0.3.29, which should allow installation with the `docker-ce` or `docker-ee` plugin.

**Comment by josegonzalez** [comment]
Closing as there is a pull request up.
